tests/acceptance/test_dbus.py

```diff
@@ -79,6 +80,53 @@ def test_dbus_system_configuration_file(self, bitbake_variables, connection):
         ).stdout.strip()
         assert "io.mender.AuthenticationManager" in output
 
+    @pytest.mark.min_mender_version("2.5.0")
+    def test_dbus_non_root_access(self, bitbake_variables, connection):
+        """Test that only root user can access Mender DBus API."""
+
+        # This is the command that is expected to fail for non-root user
+        dbus_send_command = "dbus-send --system --dest=io.mender.AuthenticationManager --print-reply /io/mender/AuthenticationManager io.mender.Authentication1.GetJwtToken"
+
+        try:
+            connection.run(
+                "cp /etc/mender/mender.conf /data/etc/mender/mender.conf.bak"
+            )
+            get_no_sftp("/etc/mender/mender.conf", connection)
+            with open("mender.conf") as fd:
+                config = json.load(fd)
+            config["DBus"] = {"Enabled": True}
+            with open("mender.conf", "w") as fd:
+                json.dump(config, fd)
+            put_no_sftp("mender.conf", connection, remote="/etc/mender/mender.conf")
```

Comment:
It's enabled by default. This entire block should be unnecessary.

Reply:
Thanks. I think it wasn't when I originally developed the test, but I will clean it up now.

Reply:
@kacf are you sure about this? On my local build (feature branches) I get:
```
# cat /etc/mender/mender.conf 
{
    "InventoryPollIntervalSeconds": 5,
    "RetryPollIntervalSeconds": 30,
    "ServerCertificate": "/usr/share/doc/mender-client/examples/demo.crt",
    "ServerURL": "https://docker.mender.io",
    "TenantToken": "dummy",
    "UpdatePollIntervalSeconds": 5
}
```

In mender source code it is disabled by default, and in meta-mender I can't find anywhere where the config option is enabled.

Reply:
> In mender source code it is disabled by default, and in meta-mender I can't find anywhere where the config option is enabled.

It is? This is a bug. It should be enabled by default if the library is compiled in. Please fix that as part of this PR, or talk to @tranchitella to have it fixed.

Reply:
I confirm it is not enabled by default, indeed.

Reply:
@kacf @tranchitella I will fix it.

Reply:
@tranchitella I don't know who @tranchitellawfp is...

Reply:
@lluiscampos sorry, my secondary github user, I posted from the wrong window/tab.

Reply:
PR: https://github.com/mendersoftware/mender/pull/676